src/components/style/themes/default.ts

```diff
@@ -0,0 +1,90 @@
+import colors from '../color/colorPalette'
+import ITheme from './themeInterface'
+import { TinyColor } from '../color/tinycolor'
+
+// Note: Define colors, fonts, and sizes by purpose.
+// Keys should not describe the value, but what they are to be used for.
+
+// Naming convention
+// [component][element][type][state][theme]
+
+const theme: ITheme = {
+  color: {
+
+    // ---
+    // BASE SCAFFOLDING
+    // ---
+    // The variables are very commonly used to designate
+    // states for components. Declaring them here and tying the
+    // component to them may make instituting new themes easier.
+
+    // Primary Color
+    primaryColor: colors.orange500,
+    get primaryColorActive () { return new TinyColor(theme.color.primaryColor).makeColor(theme.color.primaryColor, 7) },
+    get primaryColorHover () { return new TinyColor(theme.color.primaryColor).makeColor(theme.color.primaryColor, 5) },
+
+    // Accent Color
+    accentColor: colors.blurple500,
+    get accentColorActive () { return new TinyColor(theme.color.accentColor).makeColor(theme.color.accentColor, 7) },
+    get accentColorHover () { return new TinyColor(theme.color.accentColor).makeColor(theme.color.accentColor, 5) },
+
+    // Warn Color
+    warnColor: colors.red500,
+    get warnColorActive () { return new TinyColor(theme.color.warnColor).makeColor(theme.color.warnColor, 7) },
```

Comment:
This pattern means that this color manipulation function is run every time this property is accessed, which means it's running many times unneccessarily. Too many of these could cause jag.